documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -14,18 +14,21 @@
 **Note for whoever is owning the release:** please capture notes as comments in this issue for anything you noticed that could be improved for future releases.  There is a *Post Release* step below for incorporating changes back into the [RELEASE_ISSUE_TEMPLATE](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md), and this is easier done by collecting notes from along the way rather than just thinking about it at the end.
 
 First steps:
-  - [ ] FFI: Fork a new branch (`release/lotus-vX.Y.Z`) from the filecoin-ffi `master` branch
-  - [ ] FFI: Tag the head of `release/lotus-vX.Y.Z` as `vX.Y.Z-pre1`
+  - [ ] Set shell variables vX.Y.Z `X=1;Y=24;Z=3`
```

Comment:
nit, but but to be in sync with other steps include the command:

    export X=1 && export Y=24 && export Z=3

Reply:
@snadrus it looks like this is ready pending a small change from you.